icu4c/source/common/BUILD

```diff
@@ -27,13 +26,16 @@ cc_library(
         "unicode/*.h", # public
         "*.h",         # internal
         ],
+        # Instead of using this checked-in file, our Bazel build process
+        # regenerates this file and then uses the new version.
         exclude = ["norm2_nfc_data.h"],
     ),
-    # need to add includes in order to preserve existing source files'
+    # We need to add includes in order to preserve existing source files'
     # include directives that use traditional paths, not paths relative to
-    # Bazel workspace
+    # Bazel workspace:
     # https://stackoverflow.com/a/65635893/2077918
     includes = ["."],
+    # Linux platform defines.
```

Comment:
... except the one that's left here is not a Linux platform define. Please remove.